orcoursetrion/actions/github.py

```diff
@@ -38,3 +38,45 @@ def create_export_repo(course, term, description=None):
         config.ORC_STUDIO_ORG, repo_name, config.ORC_STUDIO_DEPLOY_TEAM
     )
     return repo
+
+
+def create_xml_repo(course, term, team, description=None):
```

Comment:
I see alot of this above. Would it be possible to keep things DRY by adding the team variable to `create_export_repo` set it as `None` and have it conditionally added upon its existence?